nixpkgs-src.json

```diff
@@ -1,5 +1,6 @@
 {
-    "rev":            "120b013e0c082d58a5712cde0a7371ae8b25a601",
-    "sha256":         "1hbc3ng4iy4wv0kyr1zc7n4qx3wy5qlank42zq8h87wa8z3zdf14",
-    "sha256unpacked": "0hk4y2vkgm1qadpsm4b0q1vxq889jhxzjx3ragybrlwwg54mzp4f"
+  "url": "https://github.com/NixOS/nixpkgs",
```

Comment:
shouldn't we pin our fork instead? since upstream does not have split check functionality.

Reply:
good catch. I think this was accidentally introduced in @rvl PR bumping to 18.09 in cardano-sl. This will need fixed there as well (although the way github works, it works either way).

Reply:
Surely it would only be shared if we set the nixpkgs pin in `iohk-nix`, rather than here?